src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -5234,5 +5234,30 @@ string NewMethod(object a)
 }";
             await TestAsync(code, expected, TestOptions.Script.WithLanguageVersion(LanguageVersion.CSharp7), index: CodeActionIndex);
         }
+
+        [Fact]
+        public async Task TestExtracLocalFunction_MissingInBase()
+        {
+            var code = """
+                class Base
+                {
+                    private readonly int _x;
+                    public Base(int x)
+                    {
+                        _x = x;
+                    }
+                }
+
+                class C : Base
+                {
+                    public C(int y)
+                        : base([|y + 1|])
```

Comment:
test for `this(...)` initializer as well?

Reply:
test where there's a lambda being passed upwards and someone wants to extract a local function *inside* the lambda body.